Makefile_old

```diff
@@ -0,0 +1,7 @@
+test-yes:
```

Comment:
Why do we need to keep the old makefile? Is anyone using it?

Reply:
We can remove it